packages/evolution-generator/src/tests/test_conditionals_generator.py

```
@@ -871,3 +980,168 @@ def test_non_empty_string_unchanged(self):
         """
         assert ConditionalsGenerator._empty_to_none("foo") == "foo"
         assert ConditionalsGenerator._empty_to_none(" ") == " "
+
+
+class TestPathExpansionTokensFound:
+    """Tests for ``_path_expansion_tokens_found`` (only the ``token`` key is read from each spec)."""
+
+    _SPEC_PERSON = {"token": "${currentPerson}"}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)